package.json

```diff
@@ -13,6 +13,7 @@
     "lexer"
   ],
   "main": "index.js",
+  "types": "node-tree-sitter.d.ts",
```

Comment:
How does this field get used? I'm not very familiar with typescript.

Reply:
The TypeScript compiler will automatically load types defined in the file listed there. It does this when it resolves the `import` statement. [Documentation](https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html)

Reply:
Awesome; this is a great improvement.